src/elm/Page/Community.elm

```diff
@@ -83,6 +126,7 @@ type alias Model =
     , openObjective : Maybe Int
     , modalStatus : ModalStatus
     , messageStatus : MessageStatus
+    , actions : Maybe ActionVerificationsResponse
```

Comment:
Instead of using maybe we can use a type to allow loading, what do you think?

Reply:
@lucca65 sure I considered that but I felt to avoid it since the types in this modules are so much kinda needs a rework, Our codebase has lots of standards didn't want to add more confusion

Reply:
sure thing bro, the PR is approved, please merge if you feel it is ready

Reply:
actually I'll go ahead and merge it!

Reply:
Thanks I was fast asleep